frontend/kesaseteli/employer/src/components/companyInfo/CompanyInfo.sc.ts

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+
+const CompanyInfoContainer = styled.div`
+  display: grid;
+  grid-template-columns: 17% 17% 16% 16% 17% 17%;
+  grid-template-rows: 50% 50%;
+  width: 100%;
+`;
+
+const CompanyInfoHeader = styled.div``;
+
+const CompanyInfoRow = styled.div``;
```

Comment:
Maybe we can just remove them.

Reply:
There might be later some css added, the form is now far from what was designed :)

Reply:
Fair enough